experiment_scheduler/master/grpc_master/master.proto

```diff
@@ -14,7 +14,7 @@ service Master {
 
       rpc get_task_status(Task) returns (TaskStatus) {}
 
-      rpc get_task_log(Task) returns (TaskLog) {}
+      rpc get_task_log(Task) returns (stream TaskLogFile) {}
```

Comment:
Because Log file can be large and generate log at real time, we use the stream which can transfer data continously at gRpc.